internal/kms/dummy.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 SUSE LLC.
```

Comment:
why SUSE LLC?

Reply:
I work for SUSE

Reply:
A couple thoughts:
- The copyright for everything written in and for the project is currently the "Ceph-CSI authors", for example [here](https://github.com/ceph/ceph-csi/blob/devel/cmd/cephcsi.go#L2). It's a community project, not owned by a company, so the company copyright is not expected.
- If code is copied from another project, their copyright is left in place such as [this one from Kubernetes authors](https://github.com/ceph/ceph-csi/blob/devel/internal/csi-common/nodeserver-default.go).

Reply:
Agree and above thoughts :+1: is it something you want to consider, we have someone contribution in cephcsi repo everyone uses ceph-csi authors as its a community project we dont have individual copywrite per company/contributors.